packages/client/src/hooks/useTreasury.js

```diff
@@ -50,15 +50,27 @@ const doSendFlowToTreasury = async (treasuryAddr, amount) => {
   });
 };
 
-const doProposeTransfer = async (treasuryAddr, recipientAddr, amount) => {
+const doProposeTransfer = async (
+  treasuryAddr,
+  recipientAddr,
+  amount,
+  message,
+  keyIds,
+  signatures,
+  height
+) => {
   return await mutate({
     cadence: PROPOSE_TRANSFER,
     args: (arg, t) => [
       arg(treasuryAddr, t.Address),
       arg(recipientAddr, t.Address),
       arg(amount, t.UFix64),
+      arg(message, t.String),
+      arg(keyIds, t.Array(t.UInt64)),
+      arg(signatures, t.Array(t.String)),
+      arg(height, t.UInt64),
     ],
-    limit: 55,
+    limit: 300,
```

Comment:
How do we come up with this value? Can we consider define constant to hold the limit value?

Reply:
Gas limit is used up for all these transactions that require signature validation (because there are a couple of checks needed), so I had to raise this value - first to 250 and then to 300 (because for this one this limit is not enough). There is a [question on slack](https://brud.slack.com/archives/C03ERT4F2MB/p1658363099458749) about what are the recommendations for these limits.
For the other question - yes, this can be defined as a constant - and for all the transactions that have a signature validation this limit can be the same, so I will introduce that here. Thank you for the input!

Reply:
It depends on the computationUsed - which can be read from flow emulator when the transaction is executed, for each transaction it is different, and for this one it's around 280

Reply:
Added constants